index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/a70cf7601d.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="./css/styles.css" />
+    <title>Personal Portfolio</title>
+  </head>
+
+  <body>
+    <div>
+      <header class="header">
+        <nav class="nav">
+          <a href="">Welcome</a>
+          <i class="fas fa-bars"></i>
+        </nav>
+      </header>
+      <section class="hero-heading">
+        <h1 class="title">Hey there, I am Dennis.</h1>
+        <p id="title">I'm a software Developer</p>
+        <p class="description">
+          I can help you build a product, feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <ul class="social-icons">
+          <li>
+            <a href=""><img src="./images/icons/gitHub.png" alt="" /></a>
+          </li>
+          <li>
+            <a href=""><img src="./images/icons/linkedin.png" alt="" /></a>
+          </li>
+          <li>
+            <a href=""><img src="./images/icons/angelist.png" alt="" /></a>
+          </li>
+          <li>
+            <a href=""><img src="./images/icons/twitter.png" alt="" /></a>
+          </li>
+          <li>
+            <a href=""><img src="./images/icons/medium.png" alt="" /></a>
+          </li>
+        </ul>
+      </section>
```

Comment:
- [ ] I noticed that the background of the `hero-heading` changed after you implemented some changes required, however, the new look of your page is not in line with the template you chose. The last reviewer requested for you to adjust the `header` padding since it's too much compared to your design template but these changes affected the background image. Please could you fix this? so that your work matches the design. 😄 

your page during the first review:

<img width="173" alt="chrome_ZSY8iiH3EE" src="https://user-images.githubusercontent.com/81468675/143478215-ffa1943a-f8f4-43c7-84db-70d1341515a0.png">

Your page during the second review:

<img width="173" alt="chrome_ntGbX1N3F1" src="https://user-images.githubusercontent.com/81468675/143478282-4bfab177-d648-4600-8606-b8ab64743bed.png">

Reply:
Hi, @julie-ify I am the previous reviewer. I asked him to change his background to use the mobile version as the desktop version was causing problems fitting on the screen. I'm sorry about that @denscholar. The next reviewer should approve this ✔.

Reply:
Hi, @MrBrN197 

I didn't see where you asked him to change his background image that's why I requested it. But you've explained to me that you asked him to make these changes on slack which he did. so please the next reviewer should please approve this pull request. thanks

Reply:
@MrBrN197 Thank you for the response because I was almost getting confused after spending much time working on this with your guidance. @julie-ify Thank you for the feedback. I appreciate it, I have attached the current look below.

This is my current work:

![project222](https://user-images.githubusercontent.com/48631109/143483386-56f3d2f0-8cad-4577-8a6c-dbe810ba9c80.PNG)
.